lib/widgets/home_page/simulation_test.dart

```diff
@@ -25,7 +26,7 @@ class SimulationTestWidget extends StatelessWidget {
     },
     {
       "title": "Full Test",
-      "subtitle": "140 questions",
+      "subtitle": "140 Questions",
```

Comment:
not translated, u can try :

WITH PARAMETER
in json file -> "questions" : "{} Questions"
in this line -> 'questions'.tr(args : ["140"])

WITHOUT PARAMETER (HARDCODE)
in json file -> "questions" : "140 Questions"
in this line -> 'questions'.tr()